Formula/kotlin-language-server.rb

```diff
@@ -0,0 +1,35 @@
+class KotlinLanguageServer < Formula
+  desc "Intelligent Kotlin support for any editor/IDE using the Language Server Protocol"
+  homepage "https://github.com/fwcd/kotlin-language-server"
+  url "https://github.com/fwcd/kotlin-language-server/archive/1.1.1.tar.gz"
+  sha256 "889a8866affe175c0c3fcd37b7a11e3f67b319ab3c1404d69c63b09a1796d3b4"
+  license "MIT"
+
+  bottle :unneeded
```

Comment:
Since gradle undoubtedly pulls in dependencies this should use a bottle.

Reply:
How would I go about doing this? Should I just remove `bottle :unneeded` or is there more to it?

Reply:
Yes, just remove it.